.docker/web/Dockerfile

```diff
@@ -6,17 +6,17 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends iputils-ping && \
     rm -rf /var/lib/apt/lists/*
 
-RUN curl -sL https://deb.nodesource.com/setup_18.x | bash - && \
-    apt-get install -y nodejs && \
-    npm install -g npm@latest
+# RUN curl -sL https://deb.nodesource.com/setup_18.x | bash - && \
+#     apt-get install -y nodejs && \
+#     npm install -g npm@latest
```

Comment:
Git has a history of these lines, so I don't think we need to preserve them in comments.

```suggestion
```